rasa/importers/importer.py

```diff
@@ -32,39 +32,39 @@ class TrainingFileImporter:
         use_e2e: bool = False,
         exclusion_percentage: Optional[int] = None,
     ) -> StoryGraph:
-        """Retrieves the stories which should be used for the training.
+        """Retrieves the stories that should be used for training.
 
         Args:
-            interpreter: Interpreter which should be used to parse end to
+            interpreter: Interpreter that should be used to parse end to
                          end learning annotations.
-            template_variables: Values of templates which should be replaced while
+            template_variables: Values of templates that should be replaced while
                                 reading the story files.
             use_e2e: Specifies whether to parse end to end learning annotations.
-            exclusion_percentage: Amount of training data which should be excluded.
+            exclusion_percentage: Amount of training data that should be excluded.
 
         Returns:
-            `StoryGraph` containing all loaded stories.
+            ``StoryGraph`` containing all loaded stories.
         """
 
         raise NotImplementedError()
 
-    async def get_config(self) -> Dict:
-        """"Retrieves the configuration which should be used for the training.
+    async def get_config(self) -> Dict[Text, Any]:
```

Comment:
@erohmensing This could literally be anything. So could also have some int keys.

Reply:
got it. sorry!